frontend/packages/pipelines-plugin/src/components/repository/RepositoryDetails.tsx

```diff
@@ -0,0 +1,51 @@
+import * as React from 'react';
+import { GithubIcon } from '@patternfly/react-icons';
+import { useTranslation } from 'react-i18next';
+import { SectionHeading, ResourceSummary, ExternalLink } from '@console/internal/components/utils';
+import { RepositoryKind } from './types';
+
+export interface RepositoryDetailsProps {
+  obj: RepositoryKind;
+}
+
+const RepositoryDetails: React.FC<RepositoryDetailsProps> = ({ obj: repository }) => {
+  const { t } = useTranslation();
+  const { spec } = repository;
+
+  return (
+    <div className="co-m-pane__body">
+      <SectionHeading text={t('pipelines-plugin~Repository details')} />
+      <div className="row">
+        <div className="col-sm-6">
+          <ResourceSummary resource={repository} />
+        </div>
+        {spec?.url && (
+          <div className="col-sm-6">
```

Comment:
you missed to add here.
```suggestion
          <div className="col-sm-6" data-test="pl-repository-customdetails>
```